libs/core/langchain_core/utils/pydantic.py

```diff
@@ -279,7 +279,7 @@ def _create_subset_model_v2(
     fn_description: Optional[str] = None,
 ) -> type[pydantic.BaseModel]:
     """Create a pydantic model with a subset of the model fields."""
-    from pydantic import ConfigDict, create_model
```

Comment:
same issue. ConfigDict is imported at Line: 24

Reply:
This was added in https://github.com/langchain-ai/langchain/pull/27045, when the module-level import already existed, possibly to be explicit vs. the other `_v1` function.

cc @eyurtsev